libioc/Jail.py

```diff
@@ -1802,6 +1802,7 @@ def _apply_resource_limits(
         yield event.begin()
 
         args: typing.List[str] = []
+        skipped = True
 
         if self.config['rlimits'] is False:
             yield event.skip("disabled")
```

Comment:
given the `return` below here? (git cuts it off, you'll need to unfold to see it)

Reply:
Notifying about a skipped event should be the last step of an IocEvent generator function.